pkg/data/dashboard/repo.go

```diff
@@ -73,7 +73,7 @@ func addRepos(wrangler *wrangler.Context) error {
 		return err
 	}
 
-	if features.RKE2.Enabled() {
+	if features.ProvisioningV2.Enabled() {
```

Comment:
As the fucntion basically adds repo for the specified charts. With removal of the flag, do we want addition of `rke2-charts` still conditional? Or, this is no longer relevant?

Reply:
As discussed on slack, we'll proceed with the condition check, so that these repos are added only if provisioningv2 is enabled.